tasks/December/tasks.js

```diff
@@ -0,0 +1,153 @@
+// Napisz funkcję isAdult(age), która przyjmuje wiek jako argument i zwraca true,// jeśli użytkownik ma 18 lat lub więcej, a false w przeciwnym razie.
+const isAdult = (age) => {
+	return age >= 18 ? true : false;
+};
+// 2. Stwórz funkcję findMax(a, b, c), która zwraca największą liczbę spośród trzech// podanych.
+const findMax = (a, b, c) => {
+	return Math.max(a, b, c);
+};
+// 3. Napisz funkcję isInRange(x, min, max), która sprawdza, czy liczba x mieści się// w przedziale [min, max].
+const isInRange = (x, min, max) => {
+	return x >= min && x <= max ? true : false;
+}; // 4. Stwórz funkcję getDiscount(index), która na podstawie kategorii// użytkownika (student, senior, regular) zwróci odpowiedni procent zniżki:// ● student – 20%// ● senior – 15%// ● regular – 5%
+const getDiscount = (index) => {
+	let discount = "";
+	switch (index) {
+		case "student":
+			discount = 20;
+			break;
+		case "senior":
+			discount = 15;
+			break;
+		case "regular":
+			discount = 5;
+			break;
+		default:
+			discount = 5;
+			break;
+	}
+	return discount;
```

Comment:
Tutaj mogłabyś zastosować early return

```
const getDiscount = (index) => {
	switch (index) {
		case "student":
			return 20;
		case "senior":
		         return 15;
                .....
		
	}
	
```

Wtedy nie musimy tworzyć dodatkowej zmiennej jak `discount` i nie musimy używać `break` w switch case

Reply:
Dodatkowo nazwa `index` nie jest zbyt poprawna. Pod nazwą index kryje się kategoria użytkownika, więc sugeruje użyć np. `userCategory` jako nazwa argumentu

Reply:
Tu więcej o samym Early Return Pattern https://dev.to/arikaturika/one-concept-a-day-early-return-pattern-in-javascript-3pol